scripts/ci/build.sh

```diff
@@ -176,8 +176,8 @@ fi
 ###################
 # Brotli version must match Node.js one
 # But brotli only started being shipped with Node 12
-BROTLI_NODEJS=$(node -e "console.log(process.versions.brotli || '')")
-BROTLI_DEFAULT_RELEASE=${BROTLI_NODEJS:-1.0.7}
+# BROTLI_NODEJS=$(node -e "console.log(process.versions.brotli || '')")
```

Comment:
```suggestion
BROTLI_NODEJS=$(node -e "console.log(process.versions.brotli || '')")
```
don't need to comment out this node version lookup node 18 ships with brotli 1.0.9, so its safe to leave it